package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^2.2.0-rc"
```

Comment:
What about the bug fixed in 2.5.1 (i.e., https://github.com/webpack/webpack/pull/4815)? Wouldn't the downgrade allow that to happen again?